.eleventy.js

```diff
@@ -91,6 +92,7 @@ const {
 } = require(`./${transformsDir}/service-worker-partials`);
 
 module.exports = function (config) {
+  console.log(`${chalk.black.bgGreen('Eleventy is building, please wait…')}`);
```

Comment:
This only happens once, right?

Also, super nit, the `${` doesn't really help here, we could just remove the wrapping.

Reply:
Also, not sure if you're a fan, but we use `fancy-log` in a few place which includes timestamp on the left. We could use it here too.

Reply:
Yeah, that's why it doesn't help a ton. It happens when you first run `npm run dev` but if you subsequently edit a file it won't log it again.

> Also, super nit, the ${ doesn't really help here, we could just remove the wrapping.
oh right, I was combining strings earlier and then put it all into the same message. I'll fix it.